{{cookiecutter.project_name}}/CODE_OF_CONDUCT.md

```diff
@@ -31,75 +28,68 @@ Examples of unacceptable behavior include:
 
 
-Enforcement
------------
+## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the community leaders responsible for enforcement at {{cookiecutter.email}}. All complaints will be reviewed and investigated promptly and fairly.
 
 All community leaders are obligated to respect the privacy and security of the reporter of any incident.
 
 
-Enforcement Guidelines
-----------------------
+## Enforcement Guidelines
 
 Community leaders will follow these Community Impact Guidelines in determining the consequences for any action they deem in violation of this Code of Conduct:
 
 
-1. Correction
-~~~~~~~~~~~~~
+### 1. Correction
 
 **Community Impact**: Use of inappropriate language or other behavior deemed unprofessional or unwelcome in the community.
 
 **Consequence**: A private, written warning from community leaders, providing clarity around the nature of the violation and an explanation of why the behavior was inappropriate. A public apology may be requested.
 
 
-2. Warning
-~~~~~~~~~~
+### 2. Warning
 
 **Community Impact**: A violation through a single incident or series of actions.
 
 **Consequence**: A warning with consequences for continued behavior. No interaction with the people involved, including unsolicited interaction with those enforcing the Code of Conduct, for a specified period of time. This includes avoiding interactions in community spaces as well as external channels like social media. Violating these terms may lead to a temporary or permanent ban.
 
 
-3. Temporary Ban
-~~~~~~~~~~~~~~~~
+### 3. Temporary Ban
 
 **Community Impact**: A serious violation of community standards, including sustained inappropriate behavior.
 
 **Consequence**: A temporary ban from any sort of interaction or public communication with the community for a specified period of time. No public or private interaction with the people involved, including unsolicited interaction with those enforcing the Code of Conduct, is allowed during this period. Violating these terms may lead to a permanent ban.
 
 
-4. Permanent Ban
-~~~~~~~~~~~~~~~~
+### 4. Permanent Ban
 
 **Community Impact**: Demonstrating a pattern of violation of community standards, including sustained inappropriate behavior, harassment of an individual, or aggression toward or disparagement of classes of individuals.
 
 **Consequence**: A permanent ban from any sort of public interaction within the community.
 
 
-Attribution
------------
+## Attribution
 
-This Code of Conduct is adapted from the `Contributor Covenant <homepage_>`__, version 2.0,
-available at https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+This Code of Conduct is adapted from the [Contributor Covenant][], version 2.0,
```

Comment:
Is the extra `[]` needed here (and in the next line)? I believe an earlier change flattened those.

Reply:
Thanks for your review again.

In fact, I have adopted the upstream version, and the current content is: 


## Attribution

This Code of Conduct is adapted from the [Contributor Covenant][homepage],
version 2.0, available at
[https://www.contributor-covenant.org/version/2/0/code_of_conduct.html][v2.0].


you can check [this](https://github.com/cauliyang/cookiecutter-hypermodern-python/edit/main/%7B%7Bcookiecutter.project_name%7D%7D/CODE_OF_CONDUCT.md)